src/viam/sdk/common/utils.cpp

```diff
@@ -184,5 +184,21 @@ bool from_dm_from_extra(const ProtoStruct& extra) {
     return false;
 }
 
+template <typename Src, typename Dst>
+void vecToRepeatedPtr(const std::vector<Src>& vec, google::protobuf::RepeatedPtrField<Dst>& dest) {
```

Comment:
good call -- clearing the destination in d3345726 to protect against adding to non-empty collection

I think `Assign(begin, end)` doesn't work because the types are different -- tried to get it working with constructor/operator implicit conversion approach, couldn't, guessing even harder now that the constructor/operators have been moved out of common headers